### ISSUE TITLE
Cant use audio

### ISSUE THREAD

**Issue by BlueToadMakerr** [Genymobile/scrcpy#4312]
when i try to use audio this happens
`username@penguin:~$ scrcpy --require-audio
scrcpy: unrecognized option '--require-audio'`
I'm using android 11 and ChromeOS for linux

**Comment by rom1v** [comment]
Audio is enabled by default, you don't need `--require-audio`.

The problem is your scrcpy version (`scrcpy --version`), audio was introduced in v2.0.

**Comment by miho1109** [comment]
I still can't hear anything when using Bluetooth earphones, I'm using the latest app version. Do you know why?

**Comment by Persepolite** [comment]
I'm having the same issue. Even when I disconnect from scrcpy, the phone doesn't have no audio and audio slider are greyed out

**Comment by miho1109** [comment]
I ran this command "scrcpy --no-audio" and then I could hear sound from my Bluetooth earphones again. Maybe the sound forwarding had a problem.

**Comment by rom1v** [comment]
Yes, when audio is forwarded to bluetooth, it is not forwarded to scrcpy.

It is the Android behavior when capturing the "remote submix" audio source. See #4083.

**Comment by ataur-rehman** [comment]
> I ran this command "scrcpy --no-audio" and then I could hear sound from my Bluetooth earphones again. Maybe the sound forwarding had a problem.

WOW! It works.
